src/components/App.tsx

```diff
@@ -2089,16 +2090,40 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         return;
       }
     }
-    const hitElement = this.getElementAtPosition(scenePointerX, scenePointerY);
+    document.documentElement.style.cursor = this.getCursorStyleForCurrentPointer(
+      scenePointer,
+      isOverScrollBar,
+    );
+  };
+
+  private getCursorStyleForCurrentPointer(
```

Comment:
Best to inline this one because it's not a generic-enough to warrant factoring out. Right now the `handleCanvasPointerMove` inlines half of the cursor updates, and the other half in this method.

Reply:
Thank you for the feedback. I wasn't aware of how the rest of the code was. I've inlined that.